mesheryctl/pkg/utils/helpers.go

```diff
@@ -480,8 +480,8 @@ func StringInSlice(str string, slice []string) bool {
 }
 
 // GetID returns a array of IDs from meshery server endpoint /api/{configurations}
-func GetID(configuration string) ([]string, error) {
-	url := MesheryEndpoint + "/api/" + configuration + "?page_size=10000"
+func GetID(baseUrl, configuration string) ([]string, error) {
```

Comment:
Can rename `baseUrl` to `mesheryServerUrl`

Reply:
Itemize changes that you've made to incorporate feedback and then resolve the feedback when it has been satisfied (feedback incorporated or feedback rejected), @suhail34